rules-tests/DeadCode/Rector/ClassMethod/RemoveDelegatingParentCallRector/Fixture/skip_required2.php.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveDelegatingParentCallRector\Fixture;
+
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\Service\Attribute\Required;
+
+class SkipRequired extends ParentClassWithoutRequired
+{
+    #[Required]
+    public function setEventDispatcher(EventDispatcherInterface $eventDispatcher)
+    {
+        parent::setEventDispatcher($eventDispatcher);
+    }
+}
+
+class ParentClassWithoutRequired
+{
+    public function setEventDispatcher(EventDispatcherInterface $eventDispatcher)
+    {
+    }
+}
```

Comment:
There should be only 1 class in the fixture. The extra class should be located in `/Source` directory. 
That way we also test the reflection of external class works :+1:

Reply:
updated https://github.com/rectorphp/rector-src/pull/1138/commits/34437ba5544149301edc055aa75536bd6c49b259